code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1151,6 +1151,8 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 					if(85 to INFINITY)
 						damageoverlay.overlays += brutefireloss_overlays["6"]//image("icon" = 'icons/mob/screen1_full.dmi', "icon_state" = "brutedamageoverlay6")
 				//damageoverlay.overlays += I
+
+		see_invisible = SEE_INVISIBLE_LIVING

```

Comment:
just as a heads up; see_darkness was changed here, a bit: https://github.com/ParadiseSS13/Paradise/pull/1293/files

This may or may not impact things for you.